package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "sushi-corner",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
```

Comment:
- [ ] The `start` script uses the `webpack-dev-server` dependency, but it was not added to your dev dependencies. We need to add it.

- [ ] This script does not work anymore. We need to update it.

Old
```JS
"start": "webpack-dev-server --mode development --open"
```
New
```JS
"start": "webpack serve --mode development --open"
```